### ISSUE TITLE
use Inline C; doesn't work with strict mode

### ISSUE THREAD

**Issue by akarelas-pt** [ingydotnet/inline-c-pm#69]
However, `use Inline C;` is written in the POD documentation.

**Comment by akarelas-pt** [comment]
I don't know how to make a test out of it. But here it is:

```perl
#!/usr/bin/env perl

use strict;
use warnings;

use Inline C;

__END__
__C__
void greet() {
    printf("Hello\n");
}
```

If you try to execute this perl script, you get an error:

```
Bareword "C" not allowed while "strict subs" in use at test.pl line 6.
Execution of test.pl aborted due to compilation errors.
``

**Comment by perlpunk** [comment]
Fixed by #70
